criu/crtools.c

```diff
@@ -368,9 +366,7 @@ int main(int argc, char *argv[], char *envp[])
 "                            veth[IFNAME]:OUTNAME{@BRIDGE}\n"
 "                            macvlan[IFNAME]:OUTNAME\n"
 "                            mnt[COOKIE]:ROOT\n"
-"\n"
-"  --remote              dump/restore images directly to/from remote node using\n"
-"                        image-proxy/image-cache\n"
+"  --stream              dump/restore images using criu-image-streamer\n"
```

Comment:
Could you add a description for `--stream` in `Documentation/criu.txt`?

Reply:
Yes, I forgot that one

Done